specification/evaluation-context/evaluation-context.md

```diff
@@ -1,5 +1,53 @@
 # Evaluation Context
 
+**Status**: [Experimental](../README.md#document-statuses)
+
 ## Overview
 
-// TODO: this is a stub.
+The `evaluation context` provides ambient information for the purposes of flag evaluation. Contextual data may be used as the basis for targeting, including rule-based evaluation, overrides for specific subjects, or fractional flag evaluation.
+
+### Fields
+
+See the table below for a summary of pre-defined fields.
+
+| Name          | Type   | Purpose                             | Example(s)                                                 |
+| ------------- | ------ | ----------------------------------- | ---------------------------------------------------------- |
+| targeting key | string | Subject identifier                  | "user@domain.com", "cb638b68-db72-11ec-9d64-0242ac120002"  |
+| ip            | string | Client IP address (IPv4 or IPv6)    | "216.7.132.218", "2001:0db8:85a3:0000:0000:8a2e:0370:7334" |
+| country code  | string | Subject ISO 3166-2 country code     | "BR", "US"                                                 |
+| environment   | string | Identifier for deployed environment | "prod", "development"                                      |
+| application   | string | Application identifier              | "fibonacci-api"                                            |
+| version       | string | Application version                 | "1.3.1" ,"v2-beta"                                         |
```

Comment:
this feels like something that should be set globally at the provider/client, rather that in the context for each individual evaluation.